### ISSUE TITLE
Wipe out all wp-admin styles and replace them

### ISSUE THREAD

**Issue by MichaelArestad** [MichaelArestad/Dark-Admin#1]
Ideally, we'd need between 1-3 new files to replace them all. I'm leaning on one that is heavily componentized in Sass (for max iteration speed). Ultimately, we'd go back to a straight CSS implementation at some point so we won't do too many crazy Sass things.

That said, I'm having a hell of a time figuring out how to get rid of the old styles. Here's where I'm stuck:

![image](https://cloud.githubusercontent.com/assets/1123119/9559042/801cd5e0-4daa-11e5-80ab-c2a5fb0efd2b.png)

I'm not super familiar with registering/unregistering all these things and I'm still deciphering the repurposed MP6. I'll take a stab at it tomorrow to see where I end up.

**Comment by DevinVinson** [comment]
I'm 100% game to jump in to this with as much time as I can.

Is the initial first step to completely get you back to a blank text/setup and no admin bar or do you want to build on/re-use anything already in place?

**Comment by MichaelArestad** [comment]
I'd love to get a completely unstyled canvas with no admin bar styles. Then we can re-add things one section at a time. I want to iterate on many of the current admin styles, but I think it'll be a bit more sane to start with one thing at time.

Ideally, we won't have to add any classes outside of maybe the new components added (heat map).

**Comment by MichaelArestad** [comment]
Also, thanks for your help!

**Comment by DevinVinson** [comment]
I'll poke around for a few and see if I can get that up and running.

**Comment by MichaelArestad** [comment]
@DevinVinson Just merged a PR by @egill that got rid of some of the admin bar CSS. There is definitely still some funky styling going on.

**Comment by DevinVinson** [comment]
Aside from a few errors it looks like all the styles are cleared out. The red background and a few legacy styles from mp6 are still in place from the plugin itself though.

**Comment by egill** [comment]
There's still quite a bit of .mp6 in colors-mp6.css (and other references to mp6 as well)

**Comment by DevinVinson** [comment]
The latest PR removes the references and sets up the new colors css file.

**Comment by egill** [comment]
Would love to clean the code up according to the coding standard (_cough_ camelcase _cough_). Probably better to do that sooner rather than later.

**Comment by egill** [comment]
Cleaned it up a bit and wrapped it into a class, gets a bit easier to work with.
Once you've merged the PR from @DevinVinson I'll submit my cleanup.

**Comment by egill** [comment]
A bit easier to work with IMO: https://github.com/egill/Dark-Admin/blob/master/dark-admin.php

**Comment by egill** [comment]
Don't think we need to register Open Sans on our own since it's a part of core by now.

**Comment by egill** [comment]
We don't have to register Open Sans on our own since it's now a part of core.

**Comment by egill** [comment]
Couple of things (well, three):
1. The CSS for admin-bar is incorrectly set to dark-admin.css
2. When converting to a class I ran into some issues with wp-login.php, had to "cheat" and initialize wp_styles with $wp_styles = wp_styles(); which kind of leaves a bad taste in my mouth.
That function was introduced in 4.2 (way later than the mp6 plugin), might be worth looking into that.
3. has anything been added to core since the mp6 plugin was conceived that might make things easier?

**Comment by MichaelArestad** [comment]
> We don't have to register Open Sans on our own since it's now a part of core.

True. I left that in because I'd like to register Source Sans Pro instead.

> We don't have to register Open Sans on our own since it's now a part of core.

I'll arrange a separate stylesheet for admin-bar.

**Comment by egill** [comment]
... that moment where you rebased and committed an older version of your file. Give me a minute, need to clean it up (facepalm) :)

**Comment by DevinVinson** [comment]
If you want to break things up into additional todos @MichaelArestad I'm game to try and tackle things. I'm not sure how you want to approach things so I'm open to go forward in whatever direction you've got in mind.

**Comment by egill** [comment]
^^ what he said :)

**Comment by MichaelArestad** [comment]
@DevinVinson Yep. Created an issue of things to tackle. Feel free to add or modify it: https://github.com/MichaelArestad/Dark-Admin/issues/9

**Comment by stephenwilli** [comment]
I just saw your tweet. I have always customized the wordpress admin for clients using a child admin-colors folder within the theme and then called it using this function: 

function wpt_admin_color_schemes() {

```
$theme_dir = get_stylesheet_directory_uri();

wp_admin_css_color( 
    'dubs', __( 'Dubs' ),
    $theme_dir . '/admin-colors/dubs/colors.css',
    array( '#3a3a3a', '#1f1f1f', '#528428', '#ffffff' )
);
```

}
add_action('admin_init', 'wpt_admin_color_schemes');

Then you can just select the 'Dubs' (or whatever) color scheme within the user settings. That array of colors is what shows in the user settings, and then you can find your way around the color.css from WP to make adjustments. 

I am not too familiar with creating plugins, but I assume this function can be used and you might even be able to work a color selection tool to change colors from site to site. And when Wordpress is updated, it won't overwrite that folder. 

Not sure if that helps your cause, but figure I'd share

![screen shot 2015-09-01 at 9 36 33 pm](https://cloud.githubusercontent.com/assets/7604965/9622548/8d0495fa-50f1-11e5-803d-0987f88cb8a5.png)

**Comment by ahmadawais** [comment]
@MichaelArestad Designing admin panel theme is not easy. There are several reasons to it, one of the most important is that you don't control the structure. 

E.g. some of my recent core contributions are about changing some h3 headings to h2. If you design a theme over structure like this, you'll end up with a lot of grunt work after almost every major update to WP.

**Comment by MichaelArestad** [comment]
@ahmadawais Oh I'm aware. I'm hoping we can get away with mostly stylistic changes rather than trying to change too much of the markup with a few exceptions, which we can do in self-contained modules.
